.github/workflows/release.yaml

```diff
@@ -40,3 +40,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - run: gh workflow run ci.yml --repo nextstrain/docker-base
+  create-github-release:
+    needs: [run]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.version}}
+    - name: Create GitHub Release
+      run: |
+        gh release create \
+          "${{ github.event.inputs.version}}" \
+          --repo nextstrain/augur \
+          --title "${{ github.event.inputs.version}}" \
+          --target "$(git rev-list -1 ${{ github.event.inputs.version}})" \
+          --notes "See the [official changelog](https://github.com/nextstrain/augur/blob/master/CHANGES.md) for details on what's new in this version."
```

Comment:
Hmm. I get the desire to contextualize the release within the larger changelog, but this feels quite a bit less helpful. :-/ Especially without a link to the specific version heading here, which I suppose we could add, but that starts getting messier.

Reply:
Updated with a new script `devel/changes` ported over from https://github.com/nextstrain/cli/pull/186.